modules/blocks/verify.js

```diff
@@ -38,8 +38,6 @@ __private.lastNBlockIds = [];
  * @requires async
  * @requires crypto
  * @requires lodash
- * @requires helpers/constants
```

Comment:
This is not removed uniformly across the code. Please, scan the code and remove every result.